app/graphql/mutations/profile/create.rb

```diff
@@ -12,7 +12,6 @@ class Create < BaseMutation
       argument :name, String, required: true
       argument :ref_id, ID, required: true
       argument :description, String, required: false
-      argument :business_objective_id, String, required: false
```

Comment:
Doublecheck whether FE does not use this.

Reply:
It's a duplicated argument, it's also defined as an ID and the newer version is stricter.